<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>StatusDialog</class>
 <widget class="QDialog" name="StatusDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Procedure Status</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" >
    <widget class="QTabWidget" name="tabWidget" >
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Election Information</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="label_12" >
           <property name="text" >
            <string>Question:</string>
           </property>
           <property name="buddy" >
            <cstring>questionTextEdit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="questionTextEdit" >
           <property name="frameShape" >
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow" >
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="readOnly" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="choicesLabel" >
           <property name="text" >
            <string>Choices:</string>
           </property>
           <property name="buddy" >
            <cstring></cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTreeWidget" name="choicesTreeWidget" >
           <property name="alternatingRowColors" >
            <bool>true</bool>
           </property>
           <property name="rootIsDecorated" >
            <bool>false</bool>
           </property>
           <column>
            <property name="text" >
             <string>Choice</string>
            </property>
           </column>
           <column>
            <property name="text" >
             <string>Winner</string>
            </property>
           </column>
           <column>
            <property name="text" >
             <string>Votes</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Technical Data</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="label_14" >
           <property name="text" >
            <string>Public key:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextBrowser" name="pubkeyTextBrowser" />
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>okButton</tabstop>
  <tabstop>questionTextEdit</tabstop>
 </tabstops>
 <resources>
  <include location="../resource.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>StatusDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
